src/Surfnet/ServiceProviderDashboard/Application/ViewObject/Entity.php

```diff
@@ -82,10 +82,46 @@ public static function fromEntity(DomainEntity $entity)
         );
     }
 
+    public static function fromManageResult(array $result)
+    {
+        $metadata = $result['data']['metaDataFields'];
+
+        $formattedContact = self::formatManageContact($metadata);
+
+        return new self(
+            $result['id'],
+            $result['data']['entityid'],
+            $metadata['name:en'],
+            $formattedContact,
+            'connect'
+        );
+    }
+
+    /**
+     * @return string
+     */
+    private static function formatManageContact(array $metadata)
+    {
+        for ($i=0; $i<=2; $i++) {
```

Comment:
This will work for implementation where we can save three contacts. 

In manage an additional fourth contact can be set. In theory this could result in not showing administrative contact information even though it is set.